rib/rib_test.go

```diff
@@ -1204,6 +1204,50 @@ func TestConcreteIPv4Proto(t *testing.T) {
 	}
 }
 
+func TestConcreteIPv6Proto(t *testing.T) {
```

Comment:
Is it possible to come up with a case that errors?

Reply:
I'll add a TODO here, it's difficult to figure one out because we have no error conditions that we really create, just propagating from ygot functions.